gradle.properties

```diff
@@ -3,4 +3,4 @@ version=0.1.0-SNAPSHOT
 
 # Consumed Language/Build System Versions
 javaVersion=1.8
```

Comment:
Don't forget to update chronicler/gradle/*.properties and run ./gradlew wrapper to regenerate the version used

Reply:
what do you mean?

Reply:
I do not have a properties file in my gradle folder

Reply:
If you notice, I did commit a new wrapper jar and other files. When I run gradle commands it is using the new version of gradle, not the old.

Reply:
hmmmm... run the wrapper command?

Reply:
I did run the wrapper command dude. I think the structure of the wrapper directories just changed in 4.x. Check the PR files again I think I did it right

Reply:
huh, ok then, guess they changed that